ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala

```diff
@@ -305,10 +305,18 @@ object HttpServiceTestFixture extends LazyLogging with Assertions with Inside {
       )
       .flatMap { resp =>
         val bodyF: Future[String] = getResponseDataBytes(resp, debug = true)
-        bodyF.map(body => (resp.status, body.parseJson))
+        bodyF.map(body => (resp.status, body))
       }
   }
 
+  def postJsonStringRequest(uri: Uri, jsonString: String, headers: List[HttpHeader])(
+      implicit as: ActorSystem,
+      ec: ExecutionContext,
+      mat: Materializer): Future[(StatusCode, JsValue)] =
+    postJsonStringRequestString(uri, jsonString, headers).map({
+      case (status, body) => (status, body.parseJson)
+    })
```

Comment:
Little-known Scala trick, whitespace is parentheses

```suggestion
    postJsonStringRequestString(uri, jsonString, headers).map {
      case (status, body) => (status, body.parseJson)
    }
```